website/blog/apr-2023/silk2170.md

```diff
@@ -1,169 +0,0 @@
----
```

Comment:
Please add back all the blogs (preserving URLs too). The official Docusaurus blog is a great source of inspiration, that'd be what I'd like our blog to look like. Given the official site has this, hopefully not too difficult.